<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCC::InfoFileDialog</class>
 <widget class="QDialog" name="OCC::InfoFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informations Mokapress</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="titleLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="description1Label">
      <property name="text">
       <string>Description1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="descriptionTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="description2Label">
      <property name="text">
       <string>Description2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="description2TextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="description3Label">
      <property name="text">
       <string>Description3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="description3TextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Keyword</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="keywordLineEdit"/>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OCC::InfoFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OCC::InfoFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
